moban/main.py

```diff
@@ -125,6 +126,28 @@ def handle_moban_file(moban_file, options):
     HASH_STORE.save_hashes()
 
 
+def check_none(data):
```

Comment:
At present we have a error message like `Empty value in line 2`. This is of little use if we don't have the filename.
Something like `Empty value in .moban.yml at line number 2` is good.

Reply:
sure. not a difficult job.